CONTRIBUTING.md

```diff
@@ -30,16 +30,26 @@ backwards-incompatible changes.
 
 - Ensure that GitHub Actions reports no errors.
 
-- Update the version number in `pyproject.toml`.
+- Update the version number in `typing_extensions/pyproject.toml` and in
+  `typing_extensions/CHANGELOG`.
+
+- Make sure your environment is up to date
+
+ - `git checkout master`
+ - `git pull`
+ - `python -m pip install --upgrade build twine`
 
 - Build the source and wheel distributions:
 
-  - `python -m pip install --upgrade build`
   - `cd typing_extensions`
   - `rm -rf dist/`
   - `python -m build .`
 
 - Install the built distributions locally and test (if you were using `tox`, you already
```

Comment:
Might be nice to provide the commands for this

Reply:
I tend to just install it and import it... I'd rather leave this out for now in the hope that we'll switch to fully automatic releases.